src/api/spec/factories/attribs.rb

```diff
@@ -51,7 +51,7 @@
     factory :auto_cleanup_attrib do
       attrib_type { AttribType.find_by_namespace_and_name!('OBS', 'AutoCleanup') }
 
-      values { [build(:attrib_value, value: 14)] }
+      values { [build(:attrib_value, value: (Time.now - 14.days).to_s)] }
```

Comment:
This line changes what is stored in the database and in the backend. But the test only needs what is stored in the database.

Ideally the cassettes should be also updated. There is still a '14' stored in there ([here](https://github.com/openSUSE/open-build-service/blob/b740a8ba99ec7404f914ba3a0820c919dd4f2fd1/src/api/spec/cassettes/ProjectCreateAutoCleanupRequests/_perform/with_project_without_dependencies/sets_a_deletion_request_on_the_project.yml#L50) and [here](https://github.com/openSUSE/open-build-service/blob/b740a8ba99ec7404f914ba3a0820c919dd4f2fd1/src/api/spec/cassettes/ProjectCreateAutoCleanupRequests/_perform/with_devel_package_inside_the_project/does_not_create_a_deletion_request.yml#L50)). But this works.

Reply:
You can update the cassette every 5 minutes - because as soon it's updated, the timestamp written is already outdated